core/src/executor.rs

```diff
@@ -142,6 +144,15 @@ impl Executor {
     ) -> Result<(), ValidationFail> {
         trace!("Running transaction validation");
 
+        if 0 != state_transaction.height() {
+            let Ok(authority_account) = state_transaction.world.account(authority) else {
+                return Err(ValidationFail::UnrecognizedAuthority);
+            };
+            if !authority_account.is_active {
+                return Err(ValidationFail::InactiveAuthority);
+            }
+        }
```

Comment:
this should be verified before calling executor

Reply:
Could you paraphrase, what seems like optional?